src/api/todos.ts

```diff
@@ -1,8 +1,28 @@
 import { Todo } from '../types/Todo';
 import { client } from '../utils/fetchClient';
 
-export const getTodos = (userId: number) => {
-  return client.get<Todo[]>(`/todos?userId=${userId}`);
+export const getTodos = async (userId: number) => {
+  const response = await client.get<Todo[]>(`/todos?userId=${userId}`);
+
+  return response;
 };
 
-// Add more methods here
```

Comment:
The comment 'Add more methods here' is not necessary anymore, as new methods have been added. Consider removing this comment to avoid confusion 🗑️